docs/docs/dev/building-from-source.md

```diff
@@ -53,7 +53,7 @@ open ./dist/xemu.app
     ```bash
     # Install dependencies
     sudo apt update
-    sudo apt install git build-essential cmake libsdl2-dev libepoxy-dev libpixman-1-dev libgtk-3-dev libssl-dev libsamplerate0-dev libpcap-dev ninja-build python3-yaml libslirp-dev libvulkan-dev
+    sudo apt install git build-essential cmake libsdl2-dev libepoxy-dev libpixman-1-dev libgtk-3-dev libssl-dev libsamplerate0-dev libpcap-dev ninja-build python3-pip python3-tomli python3-yaml libslirp-dev libvulkan-dev
```

Comment:
Not sure about python3-tomli, the build dependencies should match up with what's listed in the Debian packaging control file over [here](https://github.com/xemu-project/xemu/blob/672e9cd553df9fa4629ecdc49efacf310d9994b0/debian/control#L5). We should also add libcurl4-gnutls-dev while we're at it